excellent/functions/functions.go

```diff
@@ -471,14 +506,9 @@ func WordCount(env utils.Environment, text types.XText) types.XValue {
 //   @(field("a\t\tb\tc\td", 1, " ")) ->
 //   @(field("a,b,c", "foo", ",")) -> ERROR
 //
-// @function field(text, offset, delimeter)
-func Field(env utils.Environment, args ...types.XValue) types.XValue {
-	source, xerr := types.ToXText(env, args[0])
-	if xerr != nil {
-		return xerr
-	}
-
-	field, xerr := types.ToInteger(env, args[1])
+// @function field(text, offset, delimiter)
+func Field(env utils.Environment, text types.XText, args ...types.XValue) types.XValue {
```

Comment:
not sure what to do with `Field`.. if we change it to use the same logic as the other tokenizing functions then it won't parse things like `1,,3` as 3 items with the 2nd item being empty

Reply:
Ya I think field has a very specific meaning where that behavior needs to be maintained.